src/assets/svg/booster.svg

```diff
@@ -0,0 +1,11 @@
+<svg width="42" height="40" viewBox="0 0 42 40" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M3 26C3 24.8954 3.89543 24 5 24H37C38.1046 24 39 24.8954 39 26V36.2756C39 37.3802 38.1046 38.2756 37 38.2756H5C3.89543 38.2756 3 37.3802 3 36.2756V26Z" stroke="#191919" stroke-width="2"/>
+<path d="M8 29H12" stroke="#191919" stroke-width="2" stroke-linecap="round"/>
```

Comment:
i think for all strokes should be `stroke="currentColor"` not `stroke="#191919"`

Reply:
ahh, of course, fixed!